.github/workflows/lint.yml

```diff
@@ -8,32 +8,20 @@ on:
   pull_request:
 
 jobs:
-  backend:
+  lint:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
       with:
+        fetch-depth: 0
         submodules: true
-    - uses: actions/setup-node@v3
+    - uses: actions/setup-node@v3.2.0
       with:
         node-version: 18.x
-        cache: 'yarn'
-        cache-dependency-path: |
-          packages/backend/yarn.lock
-    - run: yarn install
-    - run: yarn --cwd ./packages/backend lint
+    - run: corepack enable
+    - run: yarn set version berry

```

Comment:
`corepack enable` is local to the machine.

Reply:
when corepack is enabled, corepack automatically installs `packageManager` in package.json and use it. so you shouldn't need to `yarn set version berry` because `yarn` already points to yarn berry by corepack. if `yarn set version berry` is needed, I think something is wrong.